packages/browser/src/index.ts

```diff
@@ -34,26 +48,36 @@ class Browser {
       allowInsecureUrl: true,
       apiKey,
       app: service,
-      ...(tracePropagationTargets != null
-        ? {
-            instrumentations: {
-              fetch: {
+      instrumentations: {
+        fetch: {
```

Comment:
sanity check that both `fetch` and `xhr` can map have values {}

Reply:
good call, the types say they should be able to and locally testing it looks good as well